src/Donations/LeftPanel/LeftPanelInfo.tsx

```diff
@@ -21,6 +21,7 @@ import ContactDetailsInfo from "./ContactDetailsInfo";
 interface Props {
   isMobile: boolean;
   projectDetails: FetchedProjectDetails | PlanetCashSignupDetails | null;
```

Comment:
Did you intentionally not removed `projectDetails` also to be of type `PlanetCashSignupDetails`?

Reply:
ok, fixed it.